src/cc2olx/models.py

```diff
@@ -429,12 +429,6 @@ def load_manifest_extracted(self):
         self.version = self.metadata.get("schema", {}).get("version", self.version)
         return data
 
-    def write_xml(self, text, output_base, output_file):
```

Comment:
This function wasn't being used anywhere, so choosing to remove it as opposed to updating the file encoding